.circleci/config.yml

```diff
@@ -0,0 +1,106 @@
+version: 2.1
+
+orbs:
+  slack: circleci/slack@4.2.1
+  build-tools: circleci/build-tools@2.7.1
+
+commands:
+  install_adoptopenjdk:
+    description: "Installing adoptopenjdk 8 and setting it as default Java"
+    steps:
+      - run: wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public | sudo apt-key add -
+      - run: sudo add-apt-repository --yes https://adoptopenjdk.jfrog.io/adoptopenjdk/deb/
+      - run: sudo apt update
+      - run: sudo apt install adoptopenjdk-8-hotspot
+      - run: sudo update-java-alternatives --set adoptopenjdk-8-hotspot-amd64
+
+jobs:
+  test:
+    machine:
+      image: ubuntu-2004:202010-01
+    resource_class: large
+    parameters:
+      branch-or-pr-build:
+        description: supported values are `branch` or `pr`
+        type: string
+      build-type:
+        type: string
+      build-mysql:
+        type: string
+        default: ""
+      build-mariadb:
+        type: string
+        default: ""
+      build-postgresql:
+        type: string
+        default: ""
+    steps:
+      - install_adoptopenjdk
+      - checkout
+      - when:
+          condition:
+            equal: [ pr, << parameters.branch-or-pr-build >> ]
+          steps:
+            - build-tools/merge-with-parent
+      - restore_cache:
+          key: sbt-cache
+      - run:
+          command: src/ci/bin/test.sh
+          no_output_timeout: 1h
+      - run:
+          name: Do tricks to avoid unnecessary cache updates
+          command: |
+            find ~/.ivy2/cache -name "ivydata-*.properties" -print -delete
+            find ~/.sbt -name "*.lock" -print -delete
+      - save_cache:
+          key: sbt-cache
+          paths:
+            - "~/.ivy2/cache"
+            - "~/.sbt"
+    environment:
+      CIRCLE_COMMIT_RANGE: << pipeline.git.base_revision >>...<< pipeline.git.revision >>
+      BUILD_TYPE: << parameters.build-type >>
+      BUILD_MYSQL: << parameters.build-mysql >>
+      BUILD_MARIADB: << parameters.build-mariadb >>
+      BUILD_POSTGRESQL: << parameters.build-postgresql >>
+      BUILD_BRANCH_OR_PR: << parameters.branch-or-pr-build >>
+
+workflows:
+  all-tests:
+    jobs:
+      - test:
```

Comment:
Would it be too soon to remove these test runs from `.travis.yml`?

Reply:
I think it would be better to remove Travis stuff all at once, after everything runs in CircleCI